docs/api.md

```diff
@@ -844,6 +855,7 @@ Create a new stream by passing items through the provided transducer.
 Most.js supports any transducer that implements the *de facto* JavaScript transducer protocol.  For example, two popular transducers libraries are [transducers-js](https://github.com/cognitect-labs/transducers-js) and [transducers.js](https://github.com/jlongster/transducers.js).
 
 
+<!-- skip-example -->
```

Comment:
I had assumed that transducers-js wasn't present in the repo, but I see now that it's already in dev dependencies so I've re-enabled this example. Good catch!